uk.co.compucorp.civicrm.hrleaveandabsences/scss/overrides/style/_buttons.scss

```diff
@@ -0,0 +1,14 @@
+.crm-button .crm-i {
+  top: .6em;
+  left: .6em;
+}
+
+.crm-button-type-next {
+  @include button-outline-variant(white, $brand-primary);
+  .crm-i { display: none; }
+}
+
+.crm-button-type-cancel {
+  border-color: $gray-light;
+  input { color: $text-color !important; }
```

Comment:
Please break this into 3 lines. I think CSS is easier to read when everything is consistent. JS might be another story though.

```css
input {
  color: $text-color !important;
}
```